cmake/stm32/stm32mp15xx_m4.ld

```diff
@@ -0,0 +1,202 @@
+/*
```

Comment:
for me no ST script should be part of this repo.
As you said there are licence problems and also version evolution will be problematic.
If the script cannot be generated by cmake then maybe we should just fall back to "user provides the script". There is an example for that I think.

Reply:
This file should definitely be removed and use generation done by cmake/stm32/mp1.cmake